lib/internal/process/modules.js

```diff
@@ -19,7 +23,20 @@ function normalizeReferrerURL(referrer) {
 }
 
 function initializeImportMetaObject(wrap, meta) {
-  meta.url = wrap.url;
+  const vmModule = wrapToModuleMap.get(wrap);
+  if (vmModule === undefined) {
+    // This ModuleWrap belongs to the Loader.
+    meta.url = wrap.url;
+  } else {
+    const initializeImportMeta = initImportMetaMap.get(vmModule);
+    if (initializeImportMeta === undefined) {
```

Comment:
you can change this to `if (initializeImportMeta !== undefined) { /* call method */ }`

Reply:
The reason I currently have a check for undefined is that we would like nothing to happen in the case that no callback is provided.

Reply:
i meant
```js
if (initializeImportMeta !== undefined) {
  initializeImportMeta(meta, vmModule);
} // no else, does the same thing as your code but without an empty case
```

Reply:
@punteek Did you notice the **`!==`** in @devsnek's comment?